modules/pel/peltool/peltool.py

```diff
@@ -10,7 +10,7 @@
 from pel.peltool.private_header import PrivateHeader
 from pel.peltool.user_header import UserHeader
 from pel.peltool.src import SRC
-from pel.peltool.pel_types import SectionID, SeverityValues, ActionFlagsValues
+from pel.peltool.pel_types import SectionID, SeverityValues
```

Comment:
Why are you not using ActionFlagsValues (enum)?

Reply:
As this was for hidden bit only, and we are creating a custom function in user_header, we don't need ActionFlagsValues here in peltool.py. Also to access hidden bit, added actionFlagsBits to pel_values.py.